layouts/partials/sidebar-tree.html

```diff
@@ -30,7 +30,7 @@
   </li>
   <ul>
     <li class="collapse {{ if $show }}show{{ end }}" id="{{ $sid }}">
-      {{ $pages := where (union $s.Pages $s.Sections).ByWeight ".Params.toc_hide" "!=" true }}
+      {{ $pages := where (union $s.Pages $s.Sections).ByWeight ".Params.page_hide" "!=" true }}
```

Comment:
I think we should revert this change. It's a confusing user experience to land on a page that isn't linked in the nav. If the page exists, we should show it in the nav and in the [landing page](https://deploy-preview-1707--competent-brattain-de2d6d.netlify.com/docs/started/k8s/). The page is giving useful information at this stage. When we no longer need the page, we will remove it.

Reply:
Sure @sarahmaddox, I'll revert this change.

I've made an issue to track the removal of this page: kubeflow/website#738

Reply:
Sounds good, thanks!